components/mcp230xx.rst

```diff
@@ -0,0 +1,75 @@
+MCP230xx I/O Expander
```

Comment:
Please also remove the MCP23017 .rst file then.

Reply:
Done.